404.html

```diff
@@ -30,7 +30,6 @@
     import { sampleRUM } from '/scripts/aem.js';
     sampleRUM('404', { source: document.referrer });
   </script>
-  <link rel="stylesheet" href="/styles/styles.css">
   <style>
```

Comment:
On line 41, let's add a property to the `main.error .error-number` selector to set the `fill` to `var(--color-text)` so that the "404" also updates for now.